utils/threshAlgos.py

```diff
@@ -1229,19 +1253,48 @@ def sbitRateAnalysis(chamber_config, rateTree, cutOffRate=0.0, debug=False, outf
                     pass
                 pass
             pass
+
+        # print points in a table
+        if perchannel == False:
+            print(tabulate(inflectTable, headers = ['Geo Addr', 'VFAT Number', 'ARM DAC Inflection Pt'], tablefmt='orgtbl') )
+            inflectTableFile = file("{0}/{1}/inflectionPointTable.txt".format(elogPath,chamber_config[ohKey],scandate), "w")
```

Comment:
I was hoping this file would follow the output conventions in the rest of the script.  Explicitly see [lines 1150-1160](https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/231/files#diff-b72e0013378b9ec95b76d710ee83b7c2R1150-R1160).  Apologies for not being clear earlier